handheld/console-border/shader-files/border-auto-scale.slang

```diff
@@ -8,6 +8,7 @@ layout(push_constant) uniform Push
 	float border_texture_scale;
 } params;
 
+#pragma parameter bogus_scale_instructions "[SCREEN SIZE] To change: Previous menu->LCD shader scale->Apply" 0.0 0.0 1.0 1.0
```

Comment:
Maybe it's easier to understand with "[SCREEN SIZE] In previous menu->Change LCD shader scale->Apply"

Reply:
Yeah, that sounds better.
Pushed. How does this sound now?

Reply:
It should be good yes, can't do much more than that. 👍